docs/01_nodeos/06_logging/index.md

```diff
@@ -74,13 +74,14 @@ Example:
 
 ## Loggers
 
-The logging library built into EOSIO currently supports five loggers:
+The logging library built into EOSIO currently supports the following loggers:
 
 - `default` - the default logger, always enabled.
 - `net_plugin_impl` - detailed logging for the net plugin.
 - `bnet_plugin` - detailed logging for the bnet plugin.
```

Comment:
`bnet_plugin` is no longer valid. `bnet` was removed awhile ago.

Reply:
"`http_plugin` - detailed logging for the `http_plugin`."  is missing from the list.

Reply:
Good catch - should've rechecked the list